src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepository.java

```diff
@@ -91,7 +91,8 @@ List<HearingEntity> findByCourtCodeAndHearingDay(
     @Query(value = "select * from hearing where id in (" +
         "select max(h.id) from hearing h, court_case cc where " +
         "cc.case_id = :caseId " +
-        "and h.fk_court_case_id = cc.id group by h.hearing_id)",
+        "and h.deleted = false " +
```

Comment:
do we need to add this line to all these native queries? 😱

Reply:
I know, only that one so far.